### DISCUSSION TITLE
Corn Plant - Work in progress

### DISCUSSION THREAD

**Discussion by Q4FOIL** [sunflower-land/sunflower-land#1306] [Art]
Like any good farm, corn is essential

![Schermata 2022-08-05 alle 09 24 31](https://user-images.githubusercontent.com/58088988/183024655-f4665bdc-516f-45a5-a8f2-1ace9d399c3f.png)

![Schermata 2022-08-04 alle 11 59 26](https://user-images.githubusercontent.com/58088988/182820166-4d3a546f-6c3e-424b-ae87-34cbe5ce912d.png)

Need to make first stage of plant, just need time

**Crop**

![image](https://user-images.githubusercontent.com/58088988/182818298-e1d71d36-fd5d-4963-90d9-dab971161da9.png)

![image](https://user-images.githubusercontent.com/58088988/182818359-deae8716-87ce-4c74-bfee-ece55e497a24.png)

**Seed**

![image](https://user-images.githubusercontent.com/58088988/182818433-af440781-8796-42fa-a1c9-0d5b663ab9d4.png)

![image](https://user-images.githubusercontent.com/58088988/182818496-b74aebee-3f3c-45c7-a3b0-7b33d258d93b.png)

**Plant**

_2° stage_

![image](https://user-images.githubusercontent.com/58088988/182818630-6cf0eed0-7047-4795-97ec-62a85b46da4d.png)

![image](https://user-images.githubusercontent.com/58088988/182818679-526e32dd-030c-4234-88a0-482a330517eb.png)

_Final Stage_

![image](https://user-images.githubusercontent.com/58088988/182818795-5396f6e8-116d-47a1-9553-d766bc47e781.png)

![image](https://user-images.githubusercontent.com/58088988/182818855-76909384-4189-412d-b911-e1e6b2ec8d4e.png)

_"It Ain't Much, but It's Honest Work"_

**Comment by Q4FOIL** [comment]
V2

![image](https://user-images.githubusercontent.com/58088988/183076522-7c52808c-e566-4ca4-8c7a-79e441b7764c.png)    ![image](https://user-images.githubusercontent.com/58088988/183077009-d3abcac0-5215-493e-9692-11eca4b80e16.png)     ![image](https://user-images.githubusercontent.com/58088988/183077929-2b5e4a88-aa50-4cd0-86a4-a787698022b4.png)


Big

![image](https://user-images.githubusercontent.com/58088988/183078007-037581fa-2a1a-4318-9d34-f085b8f2fde0.png)


Work in progress

**Comment by NanndoSnowzera** [comment]
very good idea, corn feeds many animals, the more food that feeds animals the better. I think there should be fusions of plants with some ingredients to form bags of animal feed. and only those who have a certain level in the EXP part of the animals can transform the plant into a feed bag, thus valuing the EXP for those who take care of the animals and leaving the food cheaper for those who have this specialty "skill" with animals. Since transforming any plant into feed it would yield more than feeding the plant itself.

**Reply by Q4FOIL** [reply]
@NanndoSnowzera 

Hi, we're going too fast!

I think for all the other animals we will have to wait, the mix function might be a good idea, but I have already read that someone are trying to create an "unatantum" booster when you need it.

If improvements are needed regarding this plant, they are available to the community

Greetings from Italy 🇮🇹 

**- Quatrefoil**
